projects/angular/src/data/datagrid/datagrid-detail.ts

```diff
@@ -27,7 +27,8 @@ import { DetailService } from './providers/detail.service';
       role="dialog"
       [id]="detailService.id"
       aria-modal="true"
-      [attr.aria-describedby]="header ? header.titleId : ''"
+      [attr.aria-labelledby]="labelledBy"
+      [attr.aria-label]="label"
```

Comment:
Only one of these should be set.

"Don't use both on the same element because aria-labelledby will take precedence over aria-label if both are applied."
 https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label#:~:text=Don%27t%20use%20both%20on,label%20if%20both%20are%20applied.

Reply:
By default `aria-labelledby` is added by the component, We need to give the user the flexibility to add an `aria-label` incase they do not want to use `aria-labelledby`. So if the user adds an `aria-label` using the input [clrDetailAriaLabel]="myLabel", then we do not set `aria-labelledby` and set only `aria-label`.

Reply:
Okay, I didn't realize one of the attributes isn't applied even though it is bound.